### ISSUE TITLE
Lack of marketing and a face people recognize as Dash representative

### ISSUE THREAD

**Issue by vampyren** [DashNetwork/dn-issues#7]
Hi, 
As a fan of Dash i think there is a lack of marketing when it comes to social media. 
Beside Bitcoin, other major projects lite ETH(Vitalik), Litecoin(Charlie), Cardano(Charles), BCH(Roger Ver), EOS(Dan) ....all have a public figure who is the face outside and people look up to. 
I have myself been around 2017 (2013 really but not to serious) + i'm in many groups and follow twitter, YouTube etc
When it comes to Dash i dont really see as much community engagement and someone who really represent Dash in the public. 
Like a great example is Charles and now his team of project managers who make videos on the progress of things. 
Also the community engagement is just huge. I want that for Dash. 
To start with we need a face outwards. Someone who can daily/weekly make tweets, videos, engage with influencers to make Dash relevant in the public eye. 
Right now Crypto is not as much fundamentals as it is about hype and excitement about new features / DeFi / NFT etc. Once we are on top 10 then maybe that can carry us but at this point with hundreds of coins/projects specially in DeFi no one really cares about great OG projects. Sad but that is the truth. Same with price. If we were to jump to 1k maybe we would be in the news but its moment 22, we need more people to be interested and buy before price can rise.

I know there are allot of things in the pipe like the platform which can be used for so much. But does anyone know about it?
Seriously doubt it. Why dont we get on YouTube with other influencers and talk about it? Tweet about all things to come continuously. Partnering with other projects to bring them onboard with their tokens and NFT. I know we are in testnet but that dont stop Cardano to do partnership (like what they done with Singularity). Bringing new to the public is a great way to show the project is alive and growing. To me it feels like we do stuff in the basement and hope people will come. NO that wont happen. Its not always the case that the best technology wins. Its about attracting users and devs. 
So with all this i would really like Dash to have 
1) One public figures who will manage the social media with continuous update about what is happening, partnership and such
2) Team/PO/PM keeping weekly/bi-weekly updates with what has been done and what will be done the coming week(s)
3) Allot more engagement from our CEO much like how Charles talks about the mission and vision and bigger picture stuff.
4) Engage the community with competition, best dapp, best article, best graphic etc etc

The common theme is to be more visible and get the name Dash more out there. Just as Amanda said in one of her video, everyone knows "Coca cola" without any subtext. But if you just say Dash most say what is that. We need to change that. 

ps. I think Amanda would be a great ambassador for the community person, not sure this is something she is interested in but it would be awesome if she wanted to tackle that role.

**Comment by awesomedash** [comment]
First DCG marketing proposal after a long time: [announcement](https://twitter.com/RTaylor05/status/1358779841001050112?s=20).

**Comment by awesomedash** [comment]
We have two (passing) marketing proposals in this cycle:
https://www.dashcentral.org/p/official-dash-newsroom-pr--communication
https://www.dashcentral.org/p/DCG-Marketing-EndFeb-EndApril

**Comment by vampyren** [comment]
Yes great progress i must say. I see way way more information out there and more activity in the social media. Maybe this topic can be on-hold for now.....

**Comment by hilawe** [comment]
There's still no "face people recognize as Dash representative" putting themselves out there on videos, interviews and other media. We don't know if or who DCG has hired their person either, nor do we know if that person is supposed to be the "face" of the network. A bit too early to put this on hold IMO.

**Comment by troydash** [comment]
> There's still no "face people recognize as Dash representative" putting themselves out there on videos, interviews and other media. We don't know if or who DCG has hired their person either, nor do we know if that person is supposed to be the "face" of the network. A bit too early to put this on hold IMO.

According to Ryan's AMA yesterday, the head of marketing they are hiring is not intended to be a "public face"

**Comment by troydash** [comment]
(Closed accidentally by misclick)

**Comment by vampyren** [comment]
> > There's still no "face people recognize as Dash representative" putting themselves out there on videos, interviews and other media. We don't know if or who DCG has hired their person either, nor do we know if that person is supposed to be the "face" of the network. A bit too early to put this on hold IMO.
> 
> According to Ryan's AMA yesterday, the head of marketing they are hiring is not intended to be a "public face"

That is strange. Who else would be that person? Its not like Dash can throw around money like crazy. Should be part of the job description i think

**Comment by jmmon** [comment]
> That is strange. Who else would be that person? Its not like Dash can throw around money like crazy. Should be part of the job description i think

That's not how it's usually done. Can you name a company who's head of marketing is known by the public or is the face of the company to the public? Of course we are a different type of "company," we could be different, but I don't think that was the plan of DCG for this role.

Joel used to be more Dash-centric on his videos but I like the way he's going with a slightly broader scope (but still "digital cash"-focused). I see Mark as the most public-facing person of Dash but he runs Dash channels instead of his own channels (good, we need those channels). I do miss Mark as the Stay Dashy hype-man ("The Dashening is a-happening!") in the Three Amigos podcasts, but even then it wasn't outward-facing.

**Comment by hilawe** [comment]
There's definitely a concerted effort by Ryan to do more Twitter outreach, AMAs, etc, lately. No doubt and it's good to see.

What we also need is the person to be doing what Ryan is doing but with a set monthly schedule of 3-4 external engagements, trying to join exclusive insider groups like Satoshi Roundtable like Evan used to be in (that's a tall task now but start somewhere), and basically become a *thought leader* in the space. No one is going to come in and become a Roger Ver or Charles Hoskinson in a month or even a year, but those two succeeded in building a following for their respective projects did it with relentless and persistent media presence. They wouldn't take 2-3 months break in between when they first got started. We should demand this IMO. There's no amount of treasury money that can substitute for a person who would do this task in the network. ABJ is not the person for this as she doesn't stick to something for too long as we all know.

**Comment by strophy** [comment]
> That's not how it's usually done.

+1 on this. Marketing is the process of bringing a product to market, it involves research and ensuring the product is a good fit for the market, as well as drumming up demand. I think most people on this thread are thinking about PR or communications roles, which are something different.

**Comment by vampyren** [comment]
> There's definitely a concerted effort by Ryan to do more Twitter outreach, AMAs, etc, lately. No doubt and it's good to see.
> 
> What we also need is the person to be doing what Ryan is doing but with a set monthly schedule of 3-4 external engagements, trying to join exclusive insider groups like Satoshi Roundtable like Evan used to be in (that's a tall task now but start somewhere), and basically become a _thought leader_ in the space. No one is going to come in and become a Roger Ver or Charles Hoskinson in a month or even a year, but those two succeeded in building a following for their respective projects did it with relentless and persistent media presence. They wouldn't take 2-3 months break in between when they first got started. We should demand this IMO. There's no amount of treasury money that can substitute for a person who would do this task in the network. ABJ is not the person for this as she doesn't stick to something for too long as we all know.

This is exactly what i'm asking too. 
And no we are not mixing the two @strophy . All we are saying is that the budget is limited and we need to have some requirement on the role. I still think the head of Marketing has to be a public figure much like Roger Ver / Charles and communicate outwards about what we are doing as @hilawe  suggested (maybe 3.4 times a month) etc....
One day we might be able to reduce this and focus more on other areas but right now i think we are on a tipping point to move down in ranking if we dont bring in more people/users and show what is happening behind the scenes. Working in the dark is a no go in crypto. 
We dont even have a good overview like this: https://cardano.ideascale.com/a/ideas/top/campaign-filter/byids/campaigns/25797/stage/unspecified?sortBy=cumulative.votes&direction=down

We have tons of things to improve i think.

**Comment by hilawe** [comment]
"Marketing is focused on promoting and selling a specific product, whereas PR is focused on maintaining a positive reputation for a company as a whole." I got you @strophy and I appreciate the need to be precise. And yes if I was to go into industry standards and definitions, communications would underlie both marketing and PR.

The head of PR's job is made much more difficult when he/she is also the one that has to promote and sell the product for the network as well as do PR because it's not being done anywhere else. Promotion of the overall vision and product(s) of the network is painfully missing.

Instead of being upset about this vicious cycle of unhappiness, we should really look at having a network-accountable CMO role outside of DCG. Both MNOs and DCG can be relieved of the tension that exists currently over this. Eventually, this CMO role can also be under the Trust Protectors (I personally prefer it stays independent of it but people disagree). MNOs will be happy that their vote will count in terms of proposal performance and DCG will be happy because they can focus on the technology and dev side. The CMO will be the bridge for all parties in the network and more importantly independent of constraints. 

Thoughts?

**Comment by vampyren** [comment]
I personally feel a disconnect between the devs / the management and community. 
Like what is being worked on even? week to week? 
Why dont any PO/PM or anyone give a brief about the internal work and progress maybe on weekly or hell bi-weekly basis? 
What is the new PR person doing? been some time and i have not heard much about it (not 100% active due to my own work so maybe i missed it but i dont see much more buzz)
Are we doing any collaboration with anyone? 
The stablecoin was very hot and then it died! Why? no one seems to be able to explain things like that to me. 
There is a big disconnect. And i think the DCG needs to start showing more presence and built confidence with the community. People generally like to help but they wont step-up if they dont even know what the heck is going on so in this case i think the team needs to step it up and start showing themselves. 
Even if this means more time is spent on communication rather then development i think its worth it. And It dont need to be the devs but maybe someone working close with them like the productowner/projectmanager or whoever. 
I'm a project manager myself and i know the importance of communication and keeping people involved and engaged...
